tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
What do you think about enabling `"noImplicitAny": true`, both here and in the main app? It seems like the only downside is that we will have to manually specify some types as `any`, (or `unknown`), with the upside being we get some additional type safety and documentation.

I imagine in the main app there would be a good chunk of places that need to get updated if we do this though, where the types are pretty relaxed.

Reply:
Good suggestion.

Somewhat related, only 16 warnings from `@typescript-eslint/no-explicit-any` rule in main app.

We within a handful from being able to change from warning to error, and then slid backwards.

Reply:
True true. Yea, I have no objections to trying it out at least in this repo

- [ ] Try enabling "noImplicitAny": true